plugin-test/src/test/groovy/org/gradle/github/dependency/extractor/SampleProjectDependencyExtractorTest.groovy

```diff
@@ -68,36 +50,15 @@ class SampleProjectDependencyExtractorTest extends BaseExtractorTest {
         succeeds("build")
 
         then:
-        resolvedConfigurations == [
-                "project :app [annotationProcessor]",
-                "project :app [classpath]",
-                "project :app [compileClasspath]",
-                "project :app [runtimeClasspath]",
-                "project :app [testAnnotationProcessor]",
-                "project :app [testCompileClasspath]",
-                "project :app [testRuntimeClasspath]",
-                "project :build-logic [annotationProcessor]",
-                "project :build-logic [classpath]",
-                "project :build-logic [compileClasspath]",
-                "project :list [annotationProcessor]",
-                "project :list [classpath]",
-                "project :list [compileClasspath]",
-                "project :utilities [annotationProcessor]",
-                "project :utilities [classpath]",
-                "project :utilities [compileClasspath]",
-                "project :utilities [testAnnotationProcessor]",
-                "project :utilities [testCompileClasspath]",
-                "project :utilities [testRuntimeClasspath]"
+        manifestNames == [
+                "project :app",
+                "project :build-logic",
+                "project :list",
+                "project :utilities"
         ]
```

Comment:
The original intention was to ensure that all of these different configurations were independently communicated. Ideally we should preserve this. Aggregating all of the results into a single entry per-project is very lossy.

For example, what happens if the project is compiled against a specific version, but the runtime classpath is set to a different version. We should be reporting all versions used in the build, just like Gradle Enterprise.

Reply:
All resolved versions are still reported, but the exact configuration that they are attached to is not. The GitHub model permits multiple versions of the same library to be reported in the same manifest.
At this time, including each resolved configuration as a separate manifest does not add any value but does add a lot of redundancy to the submitted data. Now that we have a clear mapping between the fine-grained Gradle model and the GitHub model, it will be easier to adapt this mapping if and when there is a benefit to doing so.

Reply:
> We should be reporting all versions used in the build, just like Gradle Enterprise.

Yes absolutely. I will add test coverage to confirm that this is the case.

Reply:
How will this be modeled?

Configuration A:
  - a:a:1 depends upon b:b:1

Configuration B:
  - a:a:1 depends upon b:b:1 but has a forced dependency to b:b:2